Auth/SignupLoginScreen.js

```diff
@@ -25,21 +29,37 @@ class SignupLoginScreen extends React.Component {
     this.state = {
       textUsername: '',
       textEmail: '',
-      textPassword: ''
+      textPassword: '',
+      visibleHeight: Dimensions.get('window').height
     }
   }
 
-  _goBack() {
+  componentWillMount () {
+    DeviceEventEmitter.addListener('keyboardWillShow', this.keyboardWillShow.bind(this))
+    DeviceEventEmitter.addListener('keyboardWillHide', this.keyboardWillHide.bind(this))
+  }
+
+  keyboardWillShow (e) {
+    let newSize = Dimensions.get('window').height - e.endCoordinates.height
+    this.setState({visibleHeight: newSize})
+  }
+
+  keyboardWillHide (e) {
+    this.setState({visibleHeight: Dimensions.get('window').height})
+  }
+
+  _back() {
     this.props.navigator.pop();
   }
 
   _renderBackButton() {
     return (
-      <TouchableOpacity onPress={this._goBack.bind(this)}>
-        <Image
+      <TouchableOpacity onPress={this._onPressButton}>
```

Comment:
Do we still need TouchableOpacity here? onPressButton doesn't exist and the action is handled by IconButton below.